test/cluster/storaged/01-create-sources.td

```diff
@@ -20,11 +20,15 @@ one
 > CREATE SOURCE remote1
   FROM KAFKA BROKER '${testdrive.kafka-addr}' TOPIC 'testdrive-remote1-${testdrive.seed}'
   FORMAT TEXT
-  REMOTE 'storaged:2100'
+  WITH (
+    REMOTE 'storaged:2100'
+  )
```

Comment:
I'm not sure how to run these locally, if it's possible at all, but the build is happy!

Reply:
For future reference!

```
cd test/cluster && ./mzcompose --dev run default
```